custom_components/nicehash/manifest.json

```
@@ -4,10 +4,11 @@
   "documentation": "https://github.com/brianberg/ha-nicehash",
   "issue_tracker": "https://github.com/brianberg/ha-nicehash/issues",
   "dependencies": [],
+  "version": "1.0.0",
```

Let's start at a minor version, I'm not comfortable calling this 1.0 yet
```suggestion
  "version": "0.1.0",
```